src/ripple/app/paths/cursor/DeliverNodeReverse.cpp

```diff
@@ -251,7 +252,14 @@ TER PathCursor::deliverNodeReverseImpl (
             resultCode = increment(-1).deliverNodeReverseImpl(
                 node().offerOwnerAccount_,
                 saInPassReq,
-                saInPassAct);
+                saInPassAct,
+                saOutAct > zero);
+
+            if (resultCode == tecPATH_DRY && saOutAct > zero)
+            {
+                resultCode = tesSUCCESS;
```

Comment:
To a casual reader it might be a bit surprising that a `tecPATH_DRY` translates to `tesSUCCESS`.  Consider including a comment justifying this.